src/pkg/cluster/cluster.go

```diff
@@ -44,6 +44,16 @@ const (
 	NamespaceOverrideLabel string = "zarf.dev/namespace-override"
 )
 
+// Registry TLS secret and certificate names
+const (
+	RegistryServerTLSSecret = "zarf-registry-server-tls"
+	RegistryProxyTLSSecret  = "zarf-registry-proxy-tls"
```

Comment:
This is bloating scope a little - but want to discuss. 

I believe zarf should be an enabler for applications to integrate with the zarf registry - using security protocols whenever possible. 

I would be curious about:
1. make the zarf secret name for the client information more generic
2. add future requirements that make this something we can automatically inject into zarf managed namespaces
3. Manage rolling out-of-band

Thoughts?

Reply:
Item 3 is largely covered by the mention of `zarf tools update-creds` but I was thinking about an issue to capture enhancing third-party integration. IE copy the secret into all zarf managed namespaces by default (as we do with the image pull secret for the registry) as well as an opt-in capability for rolling pods based on some annotation. 

that is all out-of-scope - what I really want to capture here is consistency. We manage secrets and certificates (as a  subset) and just want to ensure we evaluate how to be consistent in the approach where possible now or in the future (IE updating old logic).